Ginger/GingerCoreNET/ValueExpressionLib/ValueExpression.cs

```diff
@@ -96,7 +96,7 @@ public class ValueExpression : IValueExpression
         public static Regex rxEnvUrlPattern = new Regex(@"{(\bEnvURL App=)\w+\b[^{}]*}", RegexOptions.Compiled);
         
         private static Regex VBSRegex = new Regex(@"{[V|E|VBS]" + rxVar + "[^{}]*}", RegexOptions.Compiled);
-        private static Regex rxe = new Regex(@"{RegEx" + rxVare + ".*}", RegexOptions.Compiled);
+        private static Regex rxe = new Regex(@"{RegEx" + rxVare + ".*}", RegexOptions.Singleline);
```

Comment:
why "Singleline" option is insted "Compiled" option? can't we have both?
like "RegexOptions.Compiled | RegexOptions.Singleline"?

Reply:
Yes, Updated.